### ISSUE TITLE
`get_os_info` not working correctly on windows

### ISSUE THREAD

**Issue by thudjx** [lervag/vimtex#2437]
### Description

I have installed `VimTex` plugin on windows neovim. When I run `:VimtexInfo`, it seemed that `Vimtex` tried to get the system info but failed because of encoding of characters is wrong.

### Steps to reproduce

1. using `Packer` to install `VimTex`
2. `nvim test.tex`
3. `:VimtexInfo`

### Expected behavior

It returns configuration info of `VimTex`.

### Actual behavior

```
Error detected while processing function vimtex#info#open[2]..vimtex#scratch#new[2]..189[32]..191[4]..177[1]..178[1]..<SNR>94_get_os_info[15]..vimtex#jobs#cached[4]..41[5]..42:
line   10:
E474: String "<d6><f7><bb><fa><c3><fb>:           CMT" contains byte that does not start any UTF-8 character
Error detected while processing function vimtex#info#open[2]..vimtex#scratch#new[2]..189[32]..191[4]..177:
line    1:
E1098: String, List or Blob required
Press ENTER or type command to continue
```
As far as I can tell, `String "<d6><f7><bb><fa><c3><fb>:           CMT"` is the name of my PC.

### Do you use a latexmkrc file?

No

### VimtexInfo

```yaml
As stated above, it returns error info.
```

**Comment by lervag** [comment]
Huh, that's strange. I guess you are somehow using a Windows without UTF8 encoding. I don't use Windows and I don't quite know how to handle that.

What's the output of `:!systeminfo`?

Do you use Vim or neovim?

**Comment by thudjx** [comment]
You are right! I turned on UTF-8 supporting of windows accoring to [post here](https://www.tenforums.com/tutorials/132050-change-system-locale-windows-10-a.html) and solved it. I really couldn't believe UTF-8 supporting of windows is still in beta stage and is off by default...

**Comment by lervag** [comment]
Ah, cool; this means the issue is resolved, then?

**Comment by thudjx** [comment]
Sure!
